--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2023,6 +2023,7 @@
     "//chrome/browser/updates/announcement_notification",
     "//chrome/common:channel_info",
     "//chrome/common/net",
+    "//chrome/common/safe_browsing:proto",
     "//chrome/installer/util:with_no_strings",
     "//chrome/services/local_search_service:local_search_service",
     "//components/assist_ranker",
