--- a/third_party/lss/linux_syscall_support.h
+++ chromium/third_party/lss/linux_syscall_support.h
@@ -139,14 +139,17 @@
  */
 #pragma push_macro("stat64")
 #pragma push_macro("fstat64")
+#pragma push_macro("fstatat64")
 #pragma push_macro("lstat64")
 #pragma push_macro("pread64")
 #pragma push_macro("pwrite64")
 #undef stat64
 #undef fstat64
+#undef fstatat64
 #undef lstat64
 #undef pread64
 #undef pwrite64
+#define __RESTORE_MACRO 1
 
 #if defined(__ANDROID__) && defined(__x86_64__)
 // A number of x86_64 syscalls are blocked by seccomp on recent Android;
@@ -4520,9 +4523,11 @@
  * corresponding #pragma push_macro near the top of this file. */
 #pragma pop_macro("stat64")
 #pragma pop_macro("fstat64")
+#pragma pop_macro("fstatat64")
 #pragma pop_macro("lstat64")
 #pragma pop_macro("pread64")
 #pragma pop_macro("pwrite64")
+#undef __RESTORE_MACRO
 
 #if defined(__cplusplus) && !defined(SYS_CPLUSPLUS)
 }
