--- chromium-ctrl/build/config/ui.gni
+++ chromium-2/build/config/ui.gni
@@ -48,7 +48,7 @@
 assert(!use_glib || (is_linux && !is_chromeos && !is_chromecast))
 
 # Whether to use atk, the Accessibility ToolKit library
-use_atk = is_desktop_linux && !is_chromecast && use_glib
+use_atk = false
 # =============================================
 #   PLEASE DO NOT ADD MORE FLAGS TO THIS FILE
 # =============================================
