--- a/build/config/compiler/BUILD.gn
+++ chromium/build/config/compiler/BUILD.gn
@@ -403,7 +403,7 @@
   # Linux-specific compiler flags setup.
   # ------------------------------------
   if ((is_posix || is_fuchsia) && use_lld) {
-    ldflags += [ "-fuse-ld=lld" ]
+    ldflags += [ "-fuse-ld=ld" ]
     if (current_cpu == "arm64") {
       # Reduce the page size from 65536 in order to reduce binary size slightly
       # by shrinking the alignment gap between segments. This also causes all
@@ -444,7 +444,7 @@
   }
 
   if (use_icf) {
-    ldflags += [ "-Wl,--icf=all" ]
+    # ldflags += [ "-Wl,--icf=all" ]
   }
 
   if (linux_use_bundled_binutils) {
@@ -481,7 +481,7 @@
       # the driver.
       ldflags += [ "--color-diagnostics" ]
     } else {
-      ldflags += [ "-Wl,--color-diagnostics" ]
+      #ldflags += [ "-Wl,--color-diagnostics" ]
     }
   }
 
--- a/tools/v8_context_snapshot/BUILD.gn
+++ b/tools/v8_context_snapshot/BUILD.gn
@@ -85,7 +85,7 @@
     if (is_win) {
       ldflags = [ "/OPT:NOICF" ]  # link.exe, but also lld-link.exe.
     } else if (use_gold || use_lld) {
-      ldflags = [ "-Wl,--icf=none" ]
+     #ldflags = [ "-Wl,--icf=none" ]
     }
   }
 
